parts/kubernetesmasteraddons-calico-daemonset.yaml

```diff
@@ -217,8 +217,10 @@ spec:
       hostNetwork: true
       serviceAccountName: canal
       tolerations:
-        - key: node-role.kubernetes.io/master
-          effect: NoSchedule
+        - key: "node-role.kubernetes.io/master"
```

Comment:
Should we be quoting everything? These should be interpreted as strings without the quotes right?